index.js

```diff
@@ -16,6 +16,20 @@ function trbl (prefix) {
   ])
 }
 
+function logical (prefix) {
+  const rules = [prefix]
+  prefix += '-'
+
+  return rules.concat([
+    prefix + 'block',
+    prefix + 'block-start',
+    prefix + 'block-end',
+    prefix + 'inline',
+    prefix + 'inline-start',
+    prefix + 'inline-end',
```

Comment:
I believe we can argue about the order we want: this is different than the usual top right bottom left. I decided to group them by axis instead, but happy to change it if you think otherwise.